tests/cli/commands/test_cases.py

```diff
@@ -119,19 +120,19 @@ def test_case_3():
                           datetime=datetime.utcnow(),
                           properties={})
 
-        label_item.ext.enable(Extensions.LABEL)
-        label_item.ext.label.apply(label_description='ML Labels',
-                                   label_type='vector',
-                                   label_properties=['label'],
-                                   label_classes=[
-                                       LabelClasses.create(
-                                           classes=['one', 'two'],
-                                           name='label')
-                                   ],
-                                   label_tasks=['classification'],
-                                   label_methods=['manual'],
-                                   label_overviews=overviews)
-        label_item.ext.label.add_source(image_item, assets=['ortho'])
+        LabelExtension.add_to(label_item)
+        label_extension = LabelExtension.ext(label_item)
+        label_extension.apply(label_description='ML Labels',
+                              label_type='vector',
```

Comment:
I think we're encouraging folks to use the [`LabelType.VECTOR` enum](https://github.com/stac-utils/pystac/blob/7f8e7aaf29c9882e717550b219228ccf27add74b/pystac/extensions/label.py#L48) whenever possible.